.drone.yml

```diff
@@ -126,36 +126,36 @@ trigger:
 # FIMXE: 1) no need for this to be exec pipeline (can run in docker -- unjailtx doesn't ever touch enclave)
 # FIXME: 2) proper cleanup -- this thing took over 100gb of 1 day of running
 # FIXME: 3) it runs `cargo build` inside docker compose service and happily continue even if the compilation fails, wtf 
-# ---
-# kind: pipeline
-# type: exec
-# name: integration-tests-jail-unjail
-
-# platform:
-#   os: linux
-#   arch: amd64
-
-# steps:
-# - name: integration-tests
-#   commands:
-#   - cd integration-tests/jail
-#   - ./run.sh
-# - name: teardown
-#   commands:
-#   - cd integration-tests/jail
-#   - ./shutdown.sh
-#   when:
-#     status:
-#      - success
-#      - failure
-
-# trigger:
-#   branch:
-#     - master
-#     - staging
-#     - trying
-#   event:
-#     - push
+---
+kind: pipeline
+type: exec
+name: integration-tests-jail-unjail
+
+platform:
+  os: linux
+  arch: amd64
+
+steps:
+- name: integration-tests
+  commands:
+  - cd integration-tests/jail
+  - ./run.sh
+- name: teardown
+  commands:
+  - cd integration-tests/jail
+  - ./shutdown.sh
+  when:
+    status:
+     - success
+     - failure
+
+trigger:
+  branch:
+    - master
+    - staging
+    - trying
+  event:
+    - push
```

Comment:
shouldn't be re-enabled before FIXME notes are addressed -- https://github.com/crypto-com/chain/issues/853

Reply:
ok

Reply:
restored back